_sips/minutes/2020-01-27-sip-minutes.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: sips
+title: SIP Meeting Minutes - January 27 2019
+
+partof: minutes
+---
+
+# Minutes
+
+The meeting took place without a pre-defined agenda.
```

Comment:
That sounds... sad.

Maybe add something like ", but with the overall goal of reviewing progress and updating the status of features in the Scala 3 feature list (LINK)", that kind of thing? It's not like our slate was completely blank.

Reply:
plus it would be valuable for readers to summarize the items we _did_ end up talking about, so that they know whether they want to keep reading, or skip ahead to a topic that particularly interests them, that kind of thing